app/src/test/java/org/fossasia/openevent/app/presenter/EventsPresenterTest.java

```diff
@@ -135,7 +135,6 @@ public void shouldLoadOrganiserSuccessfully() {
 
         inOrder.verify(eventModel).getOrganiser(false);
         inOrder.verify(eventListView).showOrganiserName("John Wick");
-        inOrder.verify(eventListView).showOrganiserPanel(true);
```

Comment:
If so, please add a `Coordinator : ` prefix in the toolbar and make sure the string template is in strings.xml